modules/client/src/client-common/context.ts

```diff
@@ -64,12 +64,14 @@ export class Context {
       network: contextParams.network,
       signer: contextParams.signer,
       daoFactoryAddress: contextParams.daoFactoryAddress,
+      daoRegistryAddress: contextParams.daoRegistryAddress,
+      pluginRepoRegistryAddress: contextParams.pluginRepoRegistryAddress,
```

Comment:
How about `pluginRegistryAddress` ?

Reply:
It is called `pluginRepoRegistry` in the contracts so I would stick with the same name in the SDK to prevent confusions. aka Allowlist and Addresslist